src/devices/Dhtxx/samples/README.md

```diff
@@ -1,91 +1,40 @@
-# Example of DHT22
+# Example of DHTxx
 
-## Schematic
+## Hardware Required
+* DHT11/DHT12/DHT21/DHT22
+* Male/Female Jumper Wires
 
-Simply connect your DHT22 data pin to GPIO26 (physical pin 37), the ground to the ground (physical pin 6) and the VCC to +5V (physical pin 2).
+## Circuit
 
-![schema](./dht22.png)
+### 1-Wire Protocol
+Simply connect your DHTxx data pin to GPIO26 (physical pin 37), the ground to the ground (physical pin 6) and the VCC to +5V (physical pin 2).
 
-Note: if you replace the DHT22 by a DHT11, the schema will be the same.
+![schema](./dht22.png)
 
 Some sensors are already sold with the 10K resistor. Connect the GPIO26 to the *data* pin, its position can vary depending on the integrator.
 
-## Code
-
-To initialize the sensor, you need to select a pin and the type of sensor, either DHT11 or DHT22.
-
-```csharp
-DHTSensor dht = new DHTSensor(26, DhtType.Dht22);
-```
+### I2C Protocol
+![](DHT12_circuit_bb.png)
 
-You have 2 ways to get the temperature and humidity.
-
-With the first method, you need to read the sensor and then get the temperature and/or humidity:
-
-```csharp
-bool readret = dht.ReadData();
-if (readret)
-{
-    var temp = dht.Temperature;
-    var hum = dht.Humidity;
-}
-```
-
-The second way, use the ```TryGet```functions. they'll return ```true``` if a read has been successful and output the temperature and/or humidity.
-
-```csharp
-Temperature Temp;
-double Humidity;
-if (dht.TryGetTemperatureAndHumidity(out Temp, out Humidity))
-{
-    // We have Temp and Humidity and can do something
-}
-```
-
-Note: Humidity is in relative percentage air humidity. Humidity is a value between 0.0 and 100.0. 100.0 represents 100% humidity in the air.
-
-Here is a full example:
+* SCL - SCL
+* SDA - SDA
+* VCC - 5V
+* GND - GND
 
+## Code
```

Comment:
I'd recommend always putting extra enter before and after triple ticks (otherwise it sometimes breaks down markdown)